FLEDGE.md

```diff
@@ -313,7 +314,8 @@ const myAuctionConfig = {
   'trustedScoringSignalsURL': ...,
   'interestGroupBuyers': ['https://www.example-dsp.com', 'https://buyer2.com', ...],
   'auctionSignals': {...},
-  'requestedSize': {width: 100, height: 200},
+  'requestedSize': {width: '100', height: '200'},
+  'allSlotsRequestedSizedSizes': [{width: '100', height: '200'}, {width: '200', height: '300'}, ...],
```

Comment:
You meant perhaps `allSlotsRequestedSizes`?

Reply:
Thanks, Fixed!

Reply:
BTW, thank you @MattMenke2 for posting this.